src/main/java/com/volunteer/enterprise/dao/VolunteerDAOStub.java

```diff
@@ -11,10 +10,9 @@
 
 
 @Repository
-public class VolunteerServiceDAOStub implements IVolunteerServiceDAO {
+public class VolunteerDAOStub implements IVolunteerDAO {
 
     Map<Integer, Volunteer> allVolunteers = new HashMap<>();
-    Map<Integer, Company> allCompanies = new HashMap<>();
```

Comment:
Removed code meant for `CompanyDAO` class from `VolunteerDAO` class/file to separate `CompanyDAO` file.